internal/local/app.go

```diff
@@ -251,6 +271,34 @@ func (a *App) LoadConfig() error {
 	return nil
 }
 
+// LoadAppMeta will load the local app's data from .mdb/app_meta.json
+func (a *App) LoadAppMeta() error {
+	path := filepath.Join(a.RootDir, FileMdbAppMeta.String())
+	data, dataErr := ioutil.ReadFile(path)
+	if dataErr != nil {
+		return errFailedToParseAppConfig(path) // TODO: Is this the right error?
+	}
+
+	if err := json.Unmarshal(data, &a.AppMeta); err != nil {
+		return errFailedToParseAppConfig(path)
+	}
+
+	switch a.AppMeta.ConfigVersion {
+	case realm.AppConfigVersion20180301:
+		a.Config = FileStitch
+	case realm.AppConfigVersion20200603:
+		a.Config = FileConfig
+	case realm.AppConfigVersion20210101:
+		a.Config = FileRealmConfig
+	default:
+		return fmt.Errorf("invalid config version: %s", a.AppMeta.ConfigVersion.String())
+	}
+
+	a.LoadConfig()
```

Comment:
this loads the original config file, so it should be kept separate from this `LoadAppMeta` helper

Reply:
Agreed on removing the call to `LoadConfig` sinc ethat's loading a separate file. But, I do think it's worth keeping the switch statement/setting `app.Config` in this file, since this information is also being "loaded in" from app_meta.